README.md

```diff
@@ -177,7 +178,7 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
 | [Reviewpad](https://www.reviewpad.com) [:rocket:](https://reviewpad.com/discord/hiring) [:octocat:](https://github.com/reviewpad) | Pull request automations to safely merge faster with happy developers. | `Porto` <br> `Remote` |
 | [Rows](https://www.rows.com) [:rocket:](https://github.com/dashdash/hiring) | Create interactive web apps using spreadsheet skills. | `Porto` |
 | [Synopsys](https://www.synopsys.com) [:rocket:](https://synopsys.ongig.com/?l=Portugal&regions=EMEA) | Silicon chip design, verification, IP integration and application security | `Lisbon` `Porto` |
-| [Unbabel](https://unbabel.com) [:rocket:](https://unbabel.com/careers/) | Translation as a Service. | `Lisboa` |
+| [Unbabel](https://unbabel.com) [:rocket:](https://unbabel.com/careers/) [:octocat:](https://github.com/Unbabel) | Translation as a Service. | `Lisboa` |
```

Comment:
I guess there are changes here from other PRs.
Can you please check if you are rebased with the current master branch?

Reply:
added Inoweiser too

Reply:
Rebase was not done, looked like a merge from upstream which added more other commits.
I did rebase and fixed commits wording.

Reply:
@vpereira01 Thanks